### ISSUE TITLE
[Flx 2.19] Proper frame of baked rotation cannot be calculated for negative angles

### ISSUE THREAD

**Issue by goshki** [AdamAtomic/flixel#46]
I'm not sure whether it's my fault (I use Box2d to set my angles) but when a sprite with baked rotations gets negative angle then a proper frame of rotation cannot be calculated. It is so because `FlxSprite._caf` is a uint and therefore this line (421) in `FlxSprite.updateAnimation()`:

```
_caf = (angle % 360)/_bakedRotation;
```

results in an integer overflow (or rather - underflow).

The problem can be fixed by replacing the above line with:

```
var tempAngle:Number = ( angle % 360 );
if ( tempAngle < 0 ) {
    tempAngle = 360.0 - Math.abs( tempAngle ); // or just 360.0 + tempAngle :-)
}
_caf = tempAngle / _bakedRotation;
```

**Comment by AdamAtomic** [comment]
yea tru fax :(  i got lazy about that, will fix asap

**Comment by AdamAtomic** [comment]
fixed for v2.20 :)
